javascript/react-native-getting-started.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "Getting Started"
+sidebarTitle: "Getting Started"
+description: ""
+---
+
+<Note>
+  We assume you have already deployed your backend. If not, please follow 
+  [this tutorial](/outspeed_sdk/cli/deploy) to deploy your backend with `outspeed-client`.
+</Note>
+
+Let's create a mobile application using React Native to connect to the Outspeed backend. 
+If you haven't already set up a React Native (or Expo) app, you can follow the 
+[official setup guide](https://reactnative.dev/docs/environment-setup) to get started.
+
+
```

Comment:
Create an example with react-native package and add a link to the example here.

Reply:
Sure!